src/commands/editContainer/editContainer.ts

```diff
@@ -48,9 +52,42 @@ export async function editContainer(context: IActionContext, node?: ContainersIt
         managedEnvironment: await getManagedEnvironmentFromContainerApp({ ...context, ...subscriptionContext }, containerApp),
         containerApp,
         containersIdx,
+        isDraftCommand: true,
     };
     wizardContext.telemetry.properties.revisionMode = containerApp.revisionsMode;
 
+    if (isAzdExtensionInstalled()) {
```

Comment:
Why is this being added here? Doesn't seem to be related to the PR?

Reply:
Yeah, it was small enough that I snuck it in, like you said below this part is mostly boilerplate